lib/cartopy/mpl/gridliner.py

```diff
@@ -121,45 +173,75 @@ def __init__(self, axes, crs, draw_labels=False, xlocator=None,
             to determine the locations of the gridlines in the y-coordinate of
             the given CRS. Defaults to None, which implies automatic locating
             of the gridlines.
+        xformatter: optional
+            A :class:`matplotlib.ticker.Formatter` instance to format
+            longitude labels. Defaults to
+            :class:`cartopy.mpl.ticker.LongitudeFormatter`.
```

Comment:
Surely this defaults to `None`, does it not?

Reply:
Clarified